src/wallet/rpcwallet.cpp

```diff
@@ -2434,13 +2435,16 @@ static UniValue getbalances(const JSONRPCRequest& request)
 
     UniValue obj(UniValue::VOBJ);
 
+    bool avoid_reuse = wallet.IsWalletFlagSet(WALLET_FLAG_AVOID_REUSE);
     const auto bal = wallet.GetBalance();
+    const auto bal_used = avoid_reuse ? wallet.GetBalance(0, false) : bal;
     UniValue balances{UniValue::VOBJ};
     {
         UniValue balances_mine{UniValue::VOBJ};
         balances_mine.pushKV("trusted", ValueFromAmount(bal.m_mine_trusted));
         balances_mine.pushKV("untrusted_pending", ValueFromAmount(bal.m_mine_untrusted_pending));
         balances_mine.pushKV("immature", ValueFromAmount(bal.m_mine_immature));
+        if (avoid_reuse) balances_mine.pushKV("used", ValueFromAmount(bal_used.m_mine_trusted + bal_used.m_mine_untrusted_pending - bal.m_mine_trusted - bal.m_mine_untrusted_pending));
```

Comment:
7e7bb17352a3a74e142eae20572a163384fa1502

Maybe test that this is not present if `avoid_reused` flag is not set.

Reply:
Adding check to wallet_avoidreuse.py that node 0 does not include a "used" key.